### PR TITLE
chore: remove `scaffolding` from keywords

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
chore

**Did you add tests for your changes?**
NA
**If relevant, did you update the documentation?**
NA
**Summary**

Remove the `scaffolding` keyword.

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

File: packages/webpack-cli/package.json [hunk 1]
```diff
@@ -18,7 +18,6 @@
   "keywords": [
     "webpack",
     "cli",
-    "scaffolding",
```
Comment (anshumanv) [inline packages/webpack-cli/package.json:21]: we actually do scaffold things using init/loader/plugins, just not with webpack-scaffold

Comment (evenstensberg) [reply packages/webpack-cli/package.json:21]: I agree here

**Review by noharashinchan — APPROVED** [review]

File: packages/webpack-cli/package.json (hunk 1 above)
Comment (snitin315) [reply packages/webpack-cli/package.json:21]: Hmm, If we think closely it's actually another package `@webpack-cli/generators` for the scaffolding stuff. But I think we are good here. I don't have a strong opinion on this one.

Comment (rishabh3112) [reply packages/webpack-cli/package.json:21]: scaffolding is meant to be done through CLI only, so we should keep this keyword and should not be removed.

**Pull request closed without merging**